client/src/components/ProductItems/ProductItems.module.scss

```
@@ -0,0 +1,34 @@
+.ProductItems {
+	* {
+		color: black;
+		margin-left: 2%;
+		margin-right: 2%;
+	}
+	.heartIcon {
+		width: 25px;
+		padding: 7px 0 0 7px;
+	}
+	a {
+		text-decoration: none;
+	}
+
+	div {
+		text-align: center;
+	}
+
+	.itemImages {
+		width: 17vw;
+		height: 13vh;
+	}
+	.star {
+		height: 1.8vh;
+		margin-left: 15vw;
```

margin bitte nicht relativ angeben. 
```suggestion
       padding: 2rem 0;
		margin-left: 50px;
```